### PR TITLE
Add a Travis recipe

### PR DESCRIPTION
**Pull request by jimklimov**
* spellcheck text files (fixes included) - note that ATM the common nut.dict is stored in the nut source repo
* build website according to readme-documented procedure

### CODE REVIEW

File: protocols/fortress.txt [hunk 1]
```diff
@@ -1495,16 +1495,16 @@ See <<s502,section 'Automatic Startup and Shutdown (Parameters 90-93)'>>.
...
 
 |13		|*+modelname+* (*+mn+*)			|13 Model Name [Model number]		|Change not allowed			|0-16 characters			|The Fortress model number and name.
 |14		|*+serialnumber+* (*+sn+*)		|14 Serial Number [Serial number]	|Change not allowed			|0-16 characters			|Your unit's serial number.
 |15		|*+unitidentify+* (*+id+*)		|15 Unit ID				|User					|0-31 characters			|This is an identification string that you can configure for use with your network.
 |16		|*+fullload+* (*+l+*)			|16 FullLoad% 100			|Change not allowed			|Not applicable				|The percent of the Fortress' power that your equipment is using.
 |17		|*+watts+* (*+w+*)			|17 Watts 970				|Change not allowed			|Not applicable				|The Total "real power" your equipment is drawing from the Fortress.
 |18		|*+powerfactor+* (*+pf+*)		|18 PF 0.69 Dist			|Change not allowed			|Not applicable				|The power factor of your equipment.
-This is the difference in the way it draws output volta an current.
-Power factor = parameter 17 divided bu parameter 5.
+This is the difference in the way it draws output volts an current.
```
Comment (aquette) [inline protocols/fortress.txt]: volts an*d* current

Comment (jimklimov) [inline protocols/fortress.txt]: Thanks, fixed. There were too many mistakes per linear units of wording, a few probably slipped by :)

And also with ~600 exceptions added to the dictionary, mostly for driver/device keywords - but maybe some misfires too :)

**Comment by clepple** [comment]
If the intermediate commits are no longer needed, can they be squashed?

Thanks for handling the spellchecker.

**Comment by jimklimov** [comment]
Revised the stream of commits to reduce review noise

**Pull request merged**